editor/src/core/property-controls/third-party-property-controls/react-three-fiber-controls.ts

```diff
@@ -1,9 +1,15 @@
 import type { MapLike } from 'typescript'
-import { ControlDescription, ObjectControlDescription, PropertyControls } from 'utopia-api'
+import {
+  ControlDescription,
+  ObjectControlDescription,
+  Vector2ControlDescription,
+  Vector3ControlDescription,
+  PropertyControls,
+} from 'utopia-api'
 
-const Vector3: ObjectControlDescription = {
-  type: 'object',
-  object: {
+const Vector3: Vector3ControlDescription = {
+  type: 'vector3',
+  controls: {
```

Comment:
hmm, this controls field is very verbose for something which is user facing. I think it's fine if vector3 is more opinionated than this.
can the controls even be not numbers? 

Should the labels be user-settable? Or is it always x, y, z?

If it should be user configurable, then I this is how it should look like:
```javascript
const Vector3: Vector3ControlDescription = {
   type: 'vector3',
   labels: ['x', 'y', 'z'],
}
```

This should not be a "higher level" control, because it can _only_ be three numbers. Now it looks like the user can change this vector to something which has a different dimension or type – which I believe the inspector does not support because it uses the NumberControls